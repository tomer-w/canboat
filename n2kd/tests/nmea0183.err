INFO test-n2kd [n2kd] Timestamp fixed
INFO test-n2kd [n2kd] New device src=21 seen
INFO test-n2kd [n2kd] New device src=12 seen
INFO test-n2kd [n2kd] New device src=12 seen
INFO test-n2kd [n2kd] N2KD stopping
