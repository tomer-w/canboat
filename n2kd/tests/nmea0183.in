2022-09-10T12:06:58.124Z,2,129025,21,255,8,ff,ff,ff,7f,ff,ff,ff,7f
2022-09-10T12:10:33.401Z,2,129026,12,255,8,89,fc,2c,80,01,00,ff,ff
2022-09-10T12:06:58.141Z,3,129029,12,255,43,37,2d,4b,10,6d,ff,19,00,7a,5b,64,9d,56,61,07,b0,7c,b2,ca,79,db,c0,00,20,6e,bf,02,00,00,00,00,10,fc,0c,46,00,82,00,34,12,00,00,ff
