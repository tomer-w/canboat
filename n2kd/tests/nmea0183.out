$AMVTG,188.0,T,,M,0.02,N,0.04,K*7C
$AMGLL,5310.8088,N,525.7071,E,120657,A*3A
$AMRMC,120657,A,5310.8088,N,525.7071,E,0.02,188.0,100922,,,A*5B
